tests/__tests__/long-path.spec.ts

```diff
@@ -1,6 +1,16 @@
 import runJest from '../__helpers__/runJest';
+import * as fs from 'fs-extra';
 
 describe('Long path', () => {
+  const longPath =
+    'long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/long-src-path/';
+  const destinationPath = `./tests/simple-long-path/${longPath}`;
```

Comment:
should this use `__dirname`+ instead of `.`?

*so that it doesn't fail if the test is executed from a directory other than the project root?*